### PR TITLE
feat: test list component on repository page

### PR DESCRIPTION
**Pull request by madelyndubuk**
- added test list component on repo page and surface real data
Screenshot 2020-07-21 at 9 04 52 AM

### CODE REVIEW

**Comment by bcoe** [comment]
this is looking like a great start @madelyndubuk 👍 perhaps share a screen shot, or stage it and share a link when you have a moment?

File: packages/frontend/src/app/repository/repository.component.spec.ts [hunk 1]
```diff
@@ -151,7 +158,9 @@ describe('RepositoryComponent', () => {
     expect(location.path()).toEqual(expectedRoute);
   }));
 
+  /*
```
Comment (bcoe) [inline packages/frontend/src/app/repository/repository.component.spec.ts:161]: I think we could get rid of this commented out part?

File: packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts [hunk 2]
```diff
@@ -90,13 +75,34 @@ describe('TestsListComponent', () => {
...
+  it('should update the page with test data', done => {
+    COMServiceMock.fetchTests = () => of(mockTests);
+    component.ngOnInit();
+
+    const getTests = fixture.debugElement.queryAll(By.css('.test'));
+
+    setTimeout(() => {
+      expect(getTests.length).toEqual(3);
+      expect(
+        getTests[0].query(By.css('.test-name')).nativeElement.textContent
```
Comment (bcoe) [inline packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts:87]: thanks for the additional tests 👌

File: packages/frontend/src/app/repository/repository.component.ts [hunk 3]
```diff
@@ -90,8 +44,8 @@ export class RepositoryComponent implements OnInit {
 
   @ViewChild(FiltersComponent) filterComponent;
 
-  repoName = '';
-  orgName = '';
+  repoName = ' ';
+  orgName = ' ';
```
Comment (gis727) [inline packages/frontend/src/app/repository/repository.component.ts:48]: Curious to know why not the empty string 😁.

File: packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts [hunk 4]
```diff
@@ -24,47 +24,32 @@ import {AppRoutingModule} from 'src/app/routing/app-routing.module';
...
-    },
-  ];
-
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [TestsListComponent, TestDetailsComponent],
+      providers: [{provide: COMService, useValue: COMServiceMock}],
       imports: [
         AppRoutingModule,
+        HttpClientModule,
```
Comment (gis727) [inline packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts:52]: The HttpClientModule can be removed i think.

File: packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts (hunk 2 above)
Comment (gis727) [reply packages/frontend/src/app/repository/tests-list/tests-list.component.spec.ts:87]: I agree ! 🙏

File: packages/frontend/src/app/repository/repository.component.ts (hunk 3 above)
Comment (madelyndubuk) [reply packages/frontend/src/app/repository/repository.component.ts:48]: Oops! I did not mean to change that to a space.  I will revert to the empty string!

File: packages/frontend/src/app/repository/repository.component.spec.ts (hunk 1 above)
Comment (madelyndubuk) [reply packages/frontend/src/app/repository/repository.component.spec.ts:161]: Oh yes, will do!

**Comment by madelyndubuk** [comment]
Here is the link to this PR's work on the staging site: https://flaky-dev-staging.web.app/org/nodejs/node
I will add screenshots as well!
@bcoe regarding the 404 error, I will look into that and yes we should add more data to the testing database

**Pull request merged**